index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tic Tac Toe</title>
+<link rel="stylesheet" type="text/css" href="style.css"></head>
```

Comment:
Here, I would advise indenting your link tag and then put the closing </head> tag on the next line for readability, like this:

```
<head>
	<title>Tic Tac Toe</title>
    <link rel="stylesheet" type="text/css" href="style.css">
</head>
```